core/bin/zksync_api/src/fee_ticker/mod.rs

```diff
@@ -52,6 +52,8 @@ use crate::fee_ticker::{
 use crate::utils::token_db_cache::TokenDBCache;
 use zksync_types::gas_counter::GasCounter;
 
+pub use validator::types as CoinGeckoTypes;
```

Comment:
it is. the mod file exports it so that it can be used elsewhere (such as [core/bin/zksync_api/src/bin/dev-liquidity-token-watcher.rs#L15](https://github.com/rsksmart/ri-aggregation/blob/feature/dev_coingecko_watcher/core/bin/zksync_api/src/bin/dev-liquidity-token-watcher.rs#L15))